man/length_levels_linter.Rd

```diff
@@ -8,6 +8,20 @@ length_levels_linter()
 }
 \description{
 \code{length(levels(x))} is the same as \code{nlevels(x)}, but harder to read.
+}
+\examples{
```

Comment:
I wonder how that slipped...

Reply:
I must have forgotten to `roxygenize()` after adding the examples in the other PR...